components/org.wso2.carbon.database.utils/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>carbon-utils</artifactId>
+        <groupId>org.wso2.carbon.utils</groupId>
+        <version>2.0.5-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon.utils</groupId>
+    <artifactId>org.wso2.carbon.database.utils</artifactId>
```

Comment:
Why this is not a OSGi bundle ? We also need to think about how to ship this as a OSGi feature in case if this is separate from existing feature of this repo @ruwanta @ShanChathusanda93